### DISCUSSION TITLE
Separate Validation Message option for each validation rules written in Field

### DISCUSSION THREAD

**Discussion by jaybharadia** [directus/directus#18785] [Feature Requests]
### Summary

Consider a Field. I write validation rules for it. Validation rules are group of two with`and` operator. 

Problem 
- Not able to mention message for each individual rule 

Below screenshot explains the problem.

![Screenshot 2023-05-31 at 6 34 27 PM](https://github.com/directus/directus/assets/28944638/7263c746-dcae-494b-b5b4-d715401e11fb)

### Motivation

I needed to specify message for each rule separately instead of grouping the message. 

### Detailed Design

Expected

![Screenshot 2023-05-31 at 6 34 27 PM](https://github.com/directus/directus/assets/28944638/6248e7df-1ffa-4708-9421-3bf950ea9cc0)

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

No reason.

### Alternatives

N/A

### Adoption Strategy

N/A

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
